src/cider/nrepl/middleware.clj

```diff
@@ -16,6 +16,7 @@
     cider.nrepl/wrap-format
     cider.nrepl/wrap-info
     cider.nrepl/wrap-inspect
+    cider.nrepl/wrap-log
```

Comment:
@vemv I'm not sure I understand. Isn't this just the list of provided middlewares? Don't we want to always load it?

If you are concerned about performance, see my comment about the async logback logger. I think we can load it. Logs are only captured if an appender is added.

Reply:
So for the minimal Clojure API we have this one here:
https://github.com/r0man/cider-nrepl/blob/cider-log/src/cider/log/repl.clj
It's designed to be used by non-NREPL users and is used by my collegues.

But we can't use this on startup. Because the frameworks (and their state) are stored in different places. In this REPL namespace the state it bound to a var in the `cider.log.repl` namespace.

In NREPL, the framework and state are attached to the current session, so you can't use the REPL namespace for that, unless we mix those two states. I'm not sure if this approach to storing the state in the session is the right one, or if I should store it in a dynamic var as well. I took the current approach from the `inspect` middleware which saves state in the session.

Another option for stating this up automatically is to provide some hook on the ELISP side? I think it's enough to send a single NREPL message that attaches an appender to a given framework.